stl/inc/fstream

```diff
@@ -63,9 +63,9 @@ _INLINE_VAR constexpr bool _Is_any_path = _Is_any_of_v<_Ty
 _CRTIMP2_PURE FILE* __CLRCALL_PURE_OR_CDECL _Fiopen(const char*, ios_base::openmode, int);
 _CRTIMP2_PURE FILE* __CLRCALL_PURE_OR_CDECL _Fiopen(const wchar_t*, ios_base::openmode, int);
 
-#ifdef _NATIVE_WCHAR_T_DEFINED

```

Comment:
I've never given it any thought before, but seriously who put `defined` in the name of a macro so we have to query if `MEOW_DEFINED` is defined??!? This is even worse that putting `not` in the name of a boolean condition. (No change requested.)